.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
Since 2.072.2 for DMD and 1.1.0 for LDC to be correct, but I think @ximion knows this as [he made the PR](https://github.com/ldc-developers/ldc/commit/d9496882163e173ea718756a7fb0cbc627c5326f) for LDC